internal/usecase/interfaces/published.go

```diff
@@ -19,10 +19,23 @@ type ProjectPublishedMetadata struct {
 }
 
 func ProjectPublishedMetadataFrom(prj *project.Project) ProjectPublishedMetadata {
+	title := prj.PublicTitle()
+	description := prj.PublicDescription()
+	image := prj.PublicImage()
+	if title == "" {
+		title = prj.Name()
+	}
+	if description == "" {
+		description = prj.Description()
+	}
+	if image == "" {
+		image = prj.ImageURL().String()
+	}
+
```

Comment:
Whether this change is appropriate is up for debate; the reason we wanted to have a field named public was to accommodate the need for editors to keep their administrative names and images separate from their public names and images.

Rather than making this change here, it might be better to change the public field to change when the name or image is changed on the frontend side.

@HideBa How about it?